app/ssh-kss/ssh-kss.go

```diff
@@ -0,0 +1,90 @@
+// This is the ssh-keystore-server part that listens for requests of keystore-clients
+// and will sign these requests.
+package main
+
+import ()
+import (
+	"bufio"
+	"fmt"
+	"github.com/codegangsta/cli"
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/cothority/lib/ssh-ks"
+	"github.com/dedis/crypto/config"
+	"io"
+	"os"
+	"strings"
+)
+
+var serverApp *ssh_ks.ServerApp
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "SSH keystore server"
+	app.Usage = "Serves as a server to listen to requests"
+	app.Flags = []cli.Flag{
+		cli.IntFlag{
+			Name:  "debug, d",
+			Value: 1,
+			Usage: "debug-level: 1 for terse, 5 for maximal",
+		},
+		cli.StringFlag{
+			Name:  "config, c",
+			Value: "/etc/ssh-ks",
+			Usage: "The configuration-file of ssh-keystore",
+		},
+	}
+	app.Action = func(c *cli.Context) {
+		dbg.SetDebugVisible(c.Int("debug"))
+		file := c.String("config")
+		var err error
+		serverApp, err = ssh_ks.ReadServerApp(file)
+		if err != nil {
+			serverApp, err = askServerConfig(os.Stdin, os.Stdout)
+			if err != nil {
+				dbg.Fatal("While creating new config:", err)
+			}
+			err = serverApp.WriteConfig(file)
+			if err != nil {
+				dbg.Fatal("Couldn't write config:", err)
+			}
+		}
+		serverApp.Config.List()
+		err = serverApp.Start()
+		if err != nil {
+			dbg.Fatal("Couldn't start server:", err)
+		}
+		<-make(chan bool)
```

Comment:
What is this strange dark golang magic ?
Since you are also changing lib/sda/host.go, maybe we could add a Handler `CloseHandler` or a channel, `CloseChannel` which is called / used when the host is closing.
Or make the serverApp.Start() waiting.